examples/fargate-serverless/versions.tf

```diff
@@ -1,10 +1,10 @@
 terraform {
-  required_version = ">= 1.0.0"
+  required_version = ">= 1.0"
```

Comment:
just aligning versions to `major.minor` on those examples that have been updated